iree/compiler/Conversion/LinalgToSPIRV/test/pipeline_test_cooperative_mat.mlir

```diff
@@ -0,0 +1,57 @@
+// RUN: iree-opt -split-input-file -pass-pipeline="iree-codegen-linalg-to-spirv-pipeline{use-vectorization}" %s | IreeFileCheck %s
+// RUN: iree-opt -split-input-file -pass-pipeline="iree-codegen-linalg-to-spirv-pipeline{use-vectorization use-workgroup-memory}" %s | IreeFileCheck %s
+
+module attributes {
```

Comment:
Can we merge this with pipeline_test.mlir ? We can use `-split-input-file` and `-prefix-check` to consolidate all pipeline tests?

Reply:
Ah I see you moved tests from pipeline_test.mlir to this one. Sorry didnt realize that. This makes sense now.